doc/tutorial/basic/index.md

```diff
@@ -334,8 +334,12 @@ and the following to your `build.sbt`:
 jsEnv := new org.scalajs.jsenv.jsdomnodejs.JSDOMNodeJSEnv()
 {% endhighlight %}
 
-This will use the [`jsdom`](https://github.com/jsdom/jsdom) library to simulate a DOM in Node.js.
-Note that you need to install it separately using
+This will use the [`jsdom`](https://github.com/jsdom/jsdom) library to simulate a DOM in Node.js. You need to install it separately.
```

Comment:
Try to preserve one line = one sentence:
```suggestion
This will use the [`jsdom`](https://github.com/jsdom/jsdom) library to simulate a DOM in Node.js.
You need to install it separately.
```